docs/source/whitepaper.rst

```diff
@@ -390,29 +387,41 @@ stake in the challenge against Charles.
 Fees
 ~~~~
 
-Users will pay a fee for bridging their tokens. This fee needs to cover the expense of the liquidity provider and reward
-them. The fees also include a Beamer service fee, which is used for further development of the protocol.
+There are three fees that users need to pay to bridge their tokens:
+
+gas reimbursement fee
+    The fixed fee paid in source rollup's native coin that covers the liquidity
+    provider's gas cost. Collected by the agent.
+
+agent fee
+    The fee paid in token being moved, rewarding the liquidity provider. Variable and
+    currently set to 0.1% of token amount transferred. Collected by the agent.
+
+protocol fee
```

Comment:
I just realized that the protocol fee is also supposed to be derived from the transferred amount similar to the agent fee. 
For the time being the percentage is set to 0 until a DAO could decide otherwise. Could you add this to this PR please?

Reply:
> I just realized that the protocol fee is also supposed to be derived from the transferred amount similar to the agent fee. For the time being the percentage is set to 0 until a DAO could decide otherwise. Could you add this to this PR please?

Sure, I will also update the original issue description to make that clear.
Just to further clarify:

- the protocol fee is _just_ in tokens moved, there is no protocol fee in rollup's native coin, right?
- which percentage of the token amount moved should we take as the protocol fee?

Reply:
1) You are perfectly right
2) 0 percent as for now

Reply:
Reworked the PR to make the protocol fee variable too and set to 0%. Also updated the docs to reflect this.